addons/actions/functions/fnc_actions.sqf

```diff
@@ -210,53 +212,53 @@ _out pushback [localize "STR_MISERY_STITCHWOUNDS",localize "STR_MISERY_STITCHWOU
 // _out pushback [localize "STR_MISERY_REQREFUEL",localize "STR_MISERY_REQREFUEL"];
 // };
 
-if (MiseryinVehiclerepairarea == true) then {
+if (MiseryinVehiclerepairarea) then {
 _out pushback [localize "STR_MISERY_REQREPAIRS",localize "STR_MISERY_REQREPAIRS"];
 };
 
-if (MiseryinVehiclerearmarea == true) then {
+if (MiseryinVehiclerearmarea) then {
 _out pushback [localize "STR_MISERY_REQRESUPPLY",localize "STR_MISERY_REQRESUPPLY"];
 };
 
-if (MiseryinRefuelzonearea == true) then {
+if (MiseryinRefuelzonearea) then {
 _out pushback [localize "STR_MISERY_REQREFUEL",localize "STR_MISERY_REQREFUEL"];
 };
 
-if (MiseryinMedzonearea == true) then {
+if (MiseryinMedzonearea) then {
 _out pushback [localize "STR_MISERY_REQTREATMENT",localize "STR_MISERY_REQTREATMENT"];
 };
 
 //Item swappers:
 //Radiation / Geiger conversion:
 if (MiseryEnhrads) then {
-[player, "rvg_Geiger", "Misery_personalgeigernobattery"] call Misery_fnc_ItemSwap;
+    [player, "rvg_Geiger", "Misery_personalgeigernobattery"] call EFUNC(common,itemSwap);
 };
 
 //Vanilla toolkit conversion:
-[player, "ToolKit", "rvg_toolkit"] call Misery_fnc_ItemSwap;
+[player, "ToolKit", "rvg_toolkit"] call EFUNC(common,itemSwap);
 
 //ACE compat notepad swap:
 if (MiseryACE) then {
-[player, "rvg_notepad", "acex_intelitems_notepad"] call Misery_fnc_ItemSwap;
+    [player, "rvg_notepad", "acex_intelitems_notepad"] call EFUNC(common,itemSwap);
 };
```

Comment:
We can most likely yeet most rvg items, since for 2.0 it is no longer required, and I'd like to add errors if certain modules are being used from Ravage. Let me know your thoughts, we can also keep the compat, just in case ☕️